simtools/utils/general.py

```diff
@@ -316,7 +317,7 @@ def is_url(url):
     try:
         result = urlparse(url)
         return all([result.scheme, result.netloc])
-    except ValueError:
```

Comment:
This change seems to come out of nowhere (not related to the PR). In any case, it is good to test for this error in the unit test (I could not find it)

Reply:
This line has been reported by codecov to be not tested - and while adding tests it became clear that the wrong error is given in the exception statement.